public/locale/en.json

```
@@ -323,6 +323,9 @@
   "add_remarks": "Add remarks",
   "add_skill": "Add Skill",
   "add_spoke": "Add Spoke Facility",
+  "add_to_patient": "Add to Patient",
+  "add_user": "Add User",
+  "add_user_to_patient": "Add User to Patient",
```

Its more like Assining a user to the patient than adding. 
```suggestion
  "assign_user": "Assign User",
  "assign_user_to_patient": "Assign User to Patient",
```